### ISSUE TITLE
Lifted operators are already provided by NullableArrays

### ISSUE THREAD

**Issue by nalimilan** [davidanthoff/Lifts.jl#1]
So no need to provide them here:
```julia
julia> using NullableArrays

julia> Nullable(1) > Nullable(2)
Nullable{Bool}(false)
```

**Comment by davidanthoff** [comment]
Ah, cool, I wasn't aware of that! Having said that, I would prefer if lifted scalar operators were either in Base or in a separate package than NullableArrays until they can be moved into Base. Basic operations on nullable scalars have nothing to do with arrays, so it would be handy to have a package like Lifts that just covers the scalar case? Maybe that part of NullableArrays could be moved into its own package?

**Comment by davidanthoff** [comment]
Ah, and the package also doesn't cover the case of mixed ``Nullable`` and normal scalars, right? ``Nullable(1) > 3`` throws an error.

**Comment by nalimilan** [comment]
> Ah, cool, I wasn't aware of that! Having said that, I would prefer if lifted scalar operators were either in Base or in a separate package than NullableArrays until they can be moved into Base. Basic operations on nullable scalars have nothing to do with arrays, so it would be handy to have a package like Lifts that just covers the scalar case? Maybe that part of NullableArrays could be moved into its own package?

Yes, the plan is to move them into Base, but that was too late for 0.5. See discussion at https://github.com/JuliaLang/julia/pull/16988. We could have created a separate package, but it's much simpler to put everything into NullableArrays, especially for the migration path in 0.6.

> Ah, and the package also doesn't cover the case of mixed Nullable and normal scalars, right? Nullable(1) > 3 throws an error.

See https://github.com/JuliaStats/NullableArrays.jl/pull/85.
